lib/lz4hc.h

```diff
@@ -161,7 +161,13 @@ struct LZ4HC_CCtx_internal
     uint32_t   lowLimit;        /* below that point, no more dict */
     uint32_t   nextToUpdate;    /* index from which to continue dictionary update */
     short      compressionLevel;
-    short      favorDecSpeed;
+    short      flags;           /* - first 11 bits are not used currently;
```

Comment:
I think I'd rather put off the cost of doing bitflag stuff as long as possible. Can we just break
```
short favorDecSpeed;
```
into
```
int8_t favorDecSpeed;
int8_t dirty;
```
until we need to pack more in?

Reply:
Yeah, I really like this suggestion. Thanks!